lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1621,6 +1625,23 @@ public function getCreateTableSQL(Table $table, $createFlags = self::CREATE_INDE
         return array_merge($sql, $columnSql);
     }
 
+    /**
+     * @param string      $tableName
+     * @param string|null $comment
+     *
+     * @return string
+     */
+    public function getCommentOnTableSQL($tableName, $comment)
```

Comment:
Since we require php 7.2, you can remove this phpdoc and use type hints instead:
```suggestion
    public function getCommentOnTableSQL(string $tableName, ?string $comment): string
```

Reply:
Also, should this method be `final`?

Reply:
Done. I added type-hints and removed PHPDoc. Moving to PHP 7.2+ rocks :)

> Also, should this method be final? Same question for all the methods you introduced.

I reviewed all added methods and think they should not be final. Most of the methods in `AbstractPlatform` are meant to be overridden in sub-classes if some database requires a special syntax.